tests/sentry/receivers/test_onboarding.py

```diff
@@ -63,30 +62,6 @@ def test_no_existing_task(self):
         assert task.project_id == project.id
         assert task.date_completed == project.first_event
 
-    def test_existing_pending_task(self):
```

Comment:
Do we have integration tests for react_page recording the onboarding task data.

Reply:
I don't believe we do. I'll add one